src/game.h

```diff
@@ -9,6 +9,7 @@
 #include "int_id.h"
 #include "item_location.h"
 #include "cursesdef.h"
+#include "ranged.h"
```

Comment:
Would be better if you forward declared the enum and the struct, if possible.
`game.h` is included all over the place, so if `ranged.h` grows bigger, it will slow down compilation.
Plus, forward declaring instead of broad inclusions helps with refactors later on.